.github/workflows/master_release.yml

```diff
@@ -37,18 +37,27 @@ jobs:
       !(failure() || cancelled()) &&
       !contains(github.event.head_commit.message, '[skip docker_latest]') &&
       github.ref_name == 'master' && github.event_name != 'pull_request'
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
```

Comment:
Can you remind me what's the use of this?

Reply:
We need local registry as a docker container to build, pull and push multi-arch base images there. https://docs.docker.com/build/ci/github-actions/local-registry/